resources/dicts/events/misc/general.json

```diff
@@ -2079,7 +2079,7 @@
     "location": ["any"],
     "season": ["any"],
     "weight": 10,
-    "event_text":"m_c was seen chasing a rogue off of the territory.",
+      "event_text": "m_c was seen chasing a rogue from the territory.",
```

Comment:
hmm, are these really necessary?  I feel like it's easy to misread it as the rogue/kittypet being from the territory.  The previous version is much clearer on the matter.

Reply:
```suggestion
      "event_text": "m_c was seen chasing a rogue off of the territory.",
```